content/docs/user-guide/visualization/_index.md

```diff
@@ -1,8 +1,16 @@
 ---
-title: "Visualization"
-date: 2021-03-02T00:23:55-05:00
+linkTitle: "Visualization"
+title: "Visualization systems in Open 3D Engine"
+description: Learn about the various systems that provide animation and rendering in Open 3D Engine (O3DE).
 weight: 1000
 ---
 
-{{< preview-migrated >}}
+In Open 3D Engine (O3DE), you can bring actors to life with complex animated behaviors, create engaging cinematic sequences, and visualize your creations in stunning realism with O3DE's physically based renderer Atom. The topics in this section cover the concepts, tools, and processes used to achieve your creative vision.
 
+| Topic | Description |
+| - | - |
+| [Animation Editor](./animation) | With Animation Editor, you can visually construct complex animation behaviors and transitions using anim graphs. Behaviors can be driven and blended with events and states. Animation Editor also has tools you can use to create simulated objects and cloth colliders so that objects and clothing dynamically react to actor animation. |
```

Comment:
- Use "**Animation Editor**", because that is its proper name. 
- For "anim graphs": If you are talking about the tool, **Anim Graph**, then capitalize and bold the term. If you are talking about the product asset of the Anim Graph tool, animation graph, use the full term "animation graph". (For subsequent references, you can use it's shortened term, "anim graph".)
```suggestion
| [Animation Editor](./animation) | With **Animation Editor**, you can visually construct complex animation behaviors and transitions using animation graphs. Behaviors can be driven and blended with events and states. Animation Editor also has tools you can use to create simulated objects and cloth colliders so that objects and clothing dynamically react to actor animation. |
```